frontend/app/components/encryptable/form.js

```diff
@@ -134,7 +223,10 @@ export default class Form extends BaseFormComponent {
   }
 
   handleSubmitError(response) {
-    this.hasErrors = response.errors.length > 0;
+    if (response.errors.length > 0) {
+      //get error message to show correct error in form
+      this.Errors = response.errors[0].detail;
```

Comment:
This will only give you the first error though.

Reply:
Yes, I will implement multiple error handling, I did this in a rush and we were also just checking for one error before.